README.md

```diff
@@ -1 +1,24 @@
-# react-native-cli-templates
+# React Native CLI Templates
+
+These are project templates that are used by the PX Blue CLI when starting a new PX Blue project. They are used behind the scenes by the PX Blue CLI.
+
+## Available Templates
+
+### Blank
+The Blank template provides a bare-bones application with PX Blue components and themes automatically configured and ready to use. This template is available in [TypeScript](https://www.npmjs.com/package/@pxblue/react-native-template-blank-typescript)(recommended) and [JavaScript](https://www.npmjs.com/package/@pxblue/react-native-template-blank).
+
+|                | placeholder landing page | navigation drawer & routing  | login & registration screens  |
+| -------------- | -- | -- | -- |
+| Blank          | ✅ |    |    |
+
+
+## Screenshots
+##### Blank Template
+![Blank Template](./images/blank-typescript-ios.png)
```

Comment:
you may want to rename this file since it's actually a screenshot from JS, not TS. Nobody will ever know, but it's me.